libs/feature-flags/src/lib/configcat.ts

```diff
@@ -0,0 +1,28 @@
+import { FeatureFlagClient, User, FeatureFlagClientProps } from './types'
+import { createClient } from 'configcat-js'
+
+export class Client implements FeatureFlagClient {
+  private configcat: ReturnType<typeof createClient>
+
+  constructor(config: FeatureFlagClientProps) {
+    const resolvedSdkKey = config.sdkKey ?? process.env.CONFIGCAT_SDK_KEY
+    if (!resolvedSdkKey) {
+      throw new Error(
+        'Trying to initialize configcat client without CONFIGCAT_SDK_KEY environment variable',
+      )
+    }
+    if (typeof window === 'undefined') {
+      this.configcat = require('configcat-node').createClient(resolvedSdkKey)
```

Comment:
I noticed [an option called dataGovernance](https://configcat.com/docs/sdk-reference/js/#createclientwithautopollsdkkey-options), is this something we should set?

Reply:
Unless we'll be storing PII in configcat (e.g. a list of SSNs, full names, e-mails) I don't think we need it. Might be up for debate though.